packages/vx-geo/src/projections/Mercator.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import Projection from './Projection';
+
+export default function Mercator({ ...restProps }) {
```

Comment:
should have caught this in the first review but this is an unnecessary rest here.

let's go with:

```js
export default function Mercator(props) {
  return <Projection projection="mercator" {...props} />;
}
```

and well need to update the other projections. this isn't a blocker. happy to change this after a merge.